chsdi/models/vector/stopo.py

```diff
@@ -2427,3 +2427,24 @@ class SwissBuildings3dPerimeter(Base, ShopStandardClass, Vector):
     the_geom = Column(Geometry2D)
 
 register('ch.swisstopo.swissbuildings3d', SwissBuildings3dPerimeter)
+
+
+class Lotabweichungen(Base, Vector):
+    __tablename__ = 'lotabweichungen_nur_punkte'
+    __table_args__ = ({'schema': 'geodaesie', 'autoload': False})
+    __template__ = 'templates/htmlpopup/lotabweichungen.mako'
+    __bodId__ = 'ch.swisstopo.lotabweichungen'
+    __label__ = 'name'
+    id = Column('bgdi_id', Integer, primary_key=True)
```

Comment:
CONSTRAINT lotabweichungen_pkey PRIMARY KEY (id)

in DB, why not use id? It's also an integer. If you don't use ID at least change the pkey definition in DB.